.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-package-lock.json
\ No newline at end of file
+package-lock.json
+.vscode
```

Comment:
Do we want to ignore these?

- I've always committed my lock files on other projects but noticed we've done it where we omit them on some at Netlify.
- `.vscode` perhaps, but I imagine settings added might be helpful to share across the team